package.json

```diff
@@ -16,7 +16,7 @@
     "test": "npm run lint && npm run test-only",
     "test-only": "mocha -c --require test/setup.js --recursive ./test",
     "test-app": "./bin/nodejs-dashboard.js node test/app/index.js",
```

Comment:
Maybe my npm settings are incorrect.  When I put `node` in front, it works:

$ npm run test-app

> nodejs-dashboard@0.4.1 test-app C:\Users\formidable\Documents\nodejs-dashboard
> node bin/nodejs-dashboard.js node test/app/index.js

Waiting for client connection on 9838...